pom.xml

```diff
@@ -46,8 +46,8 @@
     <modules>
         <module>liberty-maven-app-parent</module>
         <module>liberty-maven-plugin</module>
-        <module>liberty-plugin-archetype</module>
+        <!--<module>liberty-plugin-archetype</module>
```

Comment:
The modules are commented out since there is no existing release to reference under the new group ID, and I don't think it makes sense to reference the old ones under wasdev.

After we do the M2 release, I can add the modules back and reference that release for the LMP version.